db/biscuits.sql

```diff
@@ -0,0 +1,71 @@
+# ************************************************************
+# Sequel Pro SQL dump
+# Version 4541
+#
+# http://www.sequelpro.com/
+# https://github.com/sequelpro/sequelpro
+#
+# Host: 127.00.0.1 (MySQL 5.7.33)
+# Database: biscuits
+# Generation Time: 2021-03-23 10:11:22 +0000
+# ************************************************************
+
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+/*!40101 SET NAMES utf8 */;
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+
+
+# Dump of table biscuits
+# ------------------------------------------------------------
+
+DROP TABLE IF EXISTS `biscuits`;
+
+CREATE TABLE `biscuits` (
+  `name` varchar(19) NOT NULL,
+  `img` varchar(181) NOT NULL,
+  `RDT` int(11) NOT NULL,
+  `description` varchar(355) NOT NULL,
+  `wikipedia` varchar(68) NOT NULL,
+  `ID` int(11) NOT NULL AUTO_INCREMENT,
```

Comment:
ID is on the end again - will appear as last column, this was already fixed in a previous commit

Reply:
resolved